CHANGELOG.md

```diff
@@ -1,4 +1,11 @@
-## 0.2.7 (Web only)
+## 0.2.7 (Common, Node.js, Web)
+
+### New features
+
+- (Node.js only) `X-ClickHouse-Summary` response header is now parsed when working with `insert`/`exec`/`command` methods.
```

Comment:
don't we need `send_progress_in_http_headers=1` header? see https://github.com/ClickHouse/clickhouse-connect/blob/6928aed5845c944007b6bf93e806313a96a93fbc/clickhouse_connect/driver/httpclient.py#L368-L370

Reply:
Interesting, something to discuss for sure. In tests, for non-streaming operations, it works regardless.

Reply:
According to the [Changelog](https://github.com/ClickHouse/ClickHouse/blob/b05a3d02ed5bbd8304bc48763a7c58b6f419eb6d/docs/changelogs/v22.6.1.1985-stable.md#improvement), it should work for non-streaming operations